src/io/report.rs

```diff
@@ -21,6 +24,9 @@ pub struct MobcoinNode {
     #[serde(skip_serializing_if = "String::is_empty")]
     pub isp: String,
     pub geo_data: GeoData,
+    pub latest_ledger: u64,
+    pub ledger_version: u32,
+    pub minimum_fee: u64,
```

Comment:
I decided to use different types because I think it's probable that `latest_ledger` may get very large some day in comparison to `ledger_version`, for example.

Reply:
While that makes sense,  I'd argue that we don't need the differentiation between unsigned integers. Given that most of the code base uses `usize`, I would prefer to stick to that for now. It may however been worth reconsidering when refactoring in the (near) future.